### ISSUE TITLE
Avoid buttons for exiting the interview

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#797]
From the VT accessibility audit:

> Some < button > elements should really be links since their function is navigation; for example, “Go to
legal aid website” if the user can’t continue the interview. We recommend changing all
elements having a navigation purpose to links.

Probably isn't anything to change in this repo, but we definitely do the same in our interviews, and probably have recommended people do the same in the docs. Making this as an umbrella issue for those other fixes, but happy to transfer it / move it around.

**Comment by BryceStevenWilley** [comment]
An important point that I'm realizing that the auditors likely misses is that by default, the "Exit" button also deletes people's answers from the server (it change depending on what exit link is set to in the metadata or default parts blocks, https://docassemble.org/docs/initial.html#metadata). We're using this default behavior, so we were deleting people's answers when they clicked the exit button. If we want to keep the behavior of deleting people's answers when they exit, we should change the exit button label to make that clear, e.g. "Exit and delete answers so far".

**Comment by nonprofittechy** [comment]
Perhaps this should be an upstream fix to Docassemble; it's a design choice Docassemble pushes even in the error message when you neglect to add a `mandatory` block to your interview.

The Weaver doesn't create any screens with an `exit` button and I don't teach it, but I think a lot of people learn to do it from reading Docassemble's documentation.

**Comment by nonprofittechy** [comment]
Closing loop on this as I did find a button that deletes answers and will address in a PR that I'll link.

Looking over this feedback, I think that the idea is to use an `<a href="">` tag and style it to look like a button for accessibility reasons. This sounds like a solidly reasoned argument but it's a change that needs to be made upstream.

**Comment by jhpyle** [comment]
Note that with the "special buttons" feature, you can use `leave` or `logout` instead of `exit`.
https://docassemble.org/docs/questions.html#action%20buttons
Note the references to `command()` and `url_of()` at the end of the section. The `url_of()` command lets you write ordinary hyperlinks in text that do things like logout.

You don't have the use the "special buttons" feature to make buttons. The `action buttons` feature is more advanced.
https://docassemble.org/docs/questions.html#special%20buttons

Also note that you can style a button so that it looks like a regular hyperlink, if that helps address the accessibility concern.

```
mandatory: True
question: |
  First screen
continue button field: first_screen
---
mandatory: True
question: |
  Final screen
subquestion: |
  This is the final screen. 

  You are done for the day.
  
  Please [go back home now](https://google.com).

  Or [log out first](${ url_of('logout', next="https://google.com") }) if you like.
action buttons:
  - label: Start over
    action: start_over
    color: info
  - label: I'll be back
    action: |
      https://google.com
    color: link
  - label: Get me out of here
    action: |
      ${ url_of('logout', next="https://google.com") }
    color: link
---
event: start_over
code: |
  command('restart')
```
